README.md

```diff
@@ -12,6 +12,8 @@
 	1. Update `appsettings.json` with the database connection string and change the Database Kind to "SqlServer".
 	1. The required schema structure will be set up automatically when starting the application for the first time.
 1. (Optional) Install [Azure Storage Emulator](https://go.microsoft.com/fwlink/?linkid=717179&clcid=0x409) to locally mock a storage account.
+    - By default, in memory stores are used.
+    -To use the Azure Storage Emulator set Storage:ConnectionString to "UseDevelopmentStorage=true;" by uncommenting it out.

```

Comment:
Nit: use `code markup` for `Storage:ConnectionString` and `UseDevelopmentStorage=true;`